assessment_software/src/app/button-component/button.component.html

```diff
@@ -5,7 +5,6 @@
   [style.background-color]="variant === 'Filled' ? background : 'transparent'"
   [style.border]="variant === 'Outlined' ? '1px solid #3056d3' : 'none'"
   [style.color]="color ? color : 'black'"
-  type="button"
```

Comment:
Keep type="button" since the component is used in lot of places. we shouldn't change the functionality because it doesn't work for specific usecase/component.